docs/contents/cloud-language.json

```diff
@@ -11,6 +11,16 @@
     }, {
         "title": "Annotation",
         "type": "language/annotation"
+    }, {
+        "title": "v1",
```

Comment:
Do you think we should standardize to this format or to the one we've used recently as illustrated [here](https://github.com/googleapis/google-cloud-php/blob/master/docs/contents/cloud-asset.json#L9-L16)? Seems odd to be moving forward with varying formats.

Reply:
Yes, we will standardize when we deprecate the manually written classes 👍